blog/2025-09-25-conda-pypi.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: "Conda ≠ PyPI: Why Conda Is More Than a Package Manager"
+slug: "conda-pypi"
+authors: [dbast, jezdez]
+tags: [conda, packaging, distribution]
+description: "Part 1 of the 'Conda Is Not PyPI' series—why conda is a multi-language user-space distribution, not just a Python package manager."
+---
+
+*Part 1 of our series “Conda Is Not PyPI: Understanding Conda as a User-Space Distro”.*
+
+<!-- truncate -->
+
+---
+
+When people first encounter conda, the most common misconception is:
+
+> *“Isn’t conda just another Python package manager, like pip?”*
+
+The short answer: **no**.
+
+Pip (and npm, RubyGems, Cargo, etc.) are **language-specific library registries**. Conda, on the other hand, is a **multi-language, user-space distribution** that can assemble coherent runtime and build environments across programming languages, system libraries, compilers, and tools.
+
+(Definition) A **user-space distribution** is a curated, relocatable filesystem prefix containing interpreters, libraries, headers, data, and tooling that runs entirely without needing privileged installation into system directories. It complements—rather than replaces—the host kernel and core C library.
+
+This first article sets the stage by exploring the difference between **library registries** and **distributions**, and why comparing conda to pip is like comparing apples to entire orchards.
```

Comment:
```suggestion
This first article sets the stage by exploring the difference between **library registries** and **distributions**, and why comparing conda to pip is like comparing an entire orchard to an apple.
```

Reply:
The ordering of the comparison should match the order conda and pip and mentioned in.